storage/ops.go

```diff
@@ -1559,15 +1574,28 @@ func FindAdvancedInstallTargets(medias []*BlockDevice) []*BlockDevice {
 						ch.FormatPartition = true
 					}
 					clrAdded = true
+					if labelMap[labelUpper] {
+						duplicateFound = true
+						log.Error("FindAdvancedInstallTargets: Found duplicate Partition label %q", label)
```

Comment:
Can we move this to a function and call it like a inner function? Because we are duplicating it

Reply:
All of this validation related code will be removed and consolidated in a validation function in the next PR.